lumapps/api/client.py

```diff
@@ -785,6 +808,52 @@ def _get_upload_url_google(self, fname, drive_id, folder_id: Optional[str]):
         upload_infos = self.get_call("document/uploadUrl/get", body=pl)
         return upload_infos["uploadUrl"]
 
+    def upload_secure_file(
+        self, name: str, f: FileIO, community_id: str
+    ) -> Optional[Dict[str, Any]]:
+        info(f"Uploading file {name}")
```

Comment:
Could you add a docstring to that method describing what it does and the args, we'll most likely make it appear in the reference doc in the future

Reply:
No not now.